system-helper/org.freedesktop.Flatpak.policy.in

```diff
@@ -235,5 +235,62 @@
     </defaults>
   </action>
 
+  <action id="org.freedesktop.Flatpak.override-parental-controls">
+    <!-- SECURITY:
+          - Authorisation to actually install software is controlled by
+            org.freedesktop.Flatpak.app-install.
+          - This action is checked after app-install, as it can only be done
+            once the app’s data (including its content rating) has been
+            downloaded.
+          - This action is checked to see if the installation should be allowed
+            based on whether the app being installed has extreme content.
```

Comment:
Maybe 'extreme' is not the best word here, since thats just one of the possible values in OARS. 

How about 'content that is not in compliance with the users parental controls profile'. If profile is the right word. What do you call the thing you store in accountsservice ?

Reply:
I agree that "extreme" is a poor word here, but lets land this and fix that in a followup.

Reply:
Attempted to be fixed in #3150.